src/Microsoft.Dnx.Tooling/ConsoleCommands/ClearCacheConsoleCommand.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+
+namespace Microsoft.Dnx.Tooling
+{
+    internal static class ClearCacheConsoleCommand
+    {
+        public static void Register(CommandLineApplication cmdApp, ReportsFactory reportsFactory)
+        {
+            cmdApp.Command("clearcache", c =>
```

Comment:
I prefer hyphenating. If we need to change other things to be consistent (eventually) so be it.